### ISSUE TITLE
Made it work in browser but not in device

### ISSUE THREAD

**Issue by balaabhinav** [akserg/ionic2-firebase-webrtc#3]
Hi Akserg, I took bits and pieces of your code and added it to my project for the webRTC video implementation using peerJS. It works great in browser but in a mobile device, it just shows the video block with a play button even after establishing connection, i am just able to hear the audio but not the video. Let me know if you have faced the same issue?
Thanks and great work!

**Comment by bolahanna44** [comment]
have u managed to find a solution for this ??

**Comment by balaabhinav** [comment]
@bolahanna44 No..Support for webRTC by certain mobile browsers are poor and another issue is with regards to android devices, marshmallow and above releases requires requesting for permission for video/audio which again adds a bottle neck.

**Comment by bolahanna44** [comment]
so what technology i should use to implement audio and video chat using ionic 3.4.0 ??

**Comment by bolahanna44** [comment]
@balaabhinav  installing crosswalk cordova plugin probably a solution i found for this

**Comment by WisaniShilumani** [comment]
@balaabhinav I installed crosswalk; I can see my own stream - but can never make calls or receive calls.
